### PR TITLE
added: open links in new tab - accidentally navigating away from current page is a pita! - also changed autofocus syntax (it does not need =true)

### PR DESCRIPTION
**Pull request by srmcgann**
added: open links in new tab - accidentally navigating away from current page is a pita! - also changed autofocus syntax (it does not need =true)

### CODE REVIEW

File: src/DweetCard.tsx [hunk 1]
```diff
@@ -497,7 +497,13 @@ export const DweetCard: React.FC<Props> = (props) => {
                 : 'none',
           }}
         >
-          <Linkify>{dweet.comments[0].text}</Linkify>
+          <Linkify componentDecorator={(decoratedHref, decoratedText, key) => (
+            <a target="blank" href={decoratedHref} key={key}>
```
Comment (lionleaf) [inline src/DweetCard.tsx:501]: For some reason I can't get this to work locally, inspecting the <a> element doesn't show a target property at all :|

Also, this <Linkify> is only for the "title" comment at the top of the dweet. I assume you'd want to add this to all comments as well, which you can do from the Linkify tag at line 603

**Review by lionleaf — CHANGES_REQUESTED** [review]
Having links from comments open in a new tab is a nice improvement, thanks!

However, I can't seem to get this to work locally, any idea what's going on? React is a bit fiddly sometimes.
Also, you've only modified the <Linkify> tag for the stickied "title"-comment at the top, not every comment.

**Comment by srmcgann** [comment]
yes. heh. "_blank" is correct. The PR has been updated.

________________________________

**Comment by lionleaf** [comment]
Have you tested this locally? I can't seem to get to work as intended. Inspecting the links just gives me a plain <a href> tag

**Review by lionleaf — APPROVED** [review]

**Comment by lionleaf** [comment]
Thanks for the implementation! I think there are still some cases that have the old behavior, but happy to get this merged in first.

**Pull request merged**